recipes/jemalloc/all/conanfile.py

```diff
@@ -114,111 +137,51 @@ def _autotools_args(self):
             "--enable-log" if self.options.enable_debug_logging else "--disable-log",
             "--enable-initial-exec-tls" if self.options.enable_initial_exec_tls else "--disable-initial-exec-tls",
             "--enable-libdl" if self.options.enable_libdl else "--disable-libdl",
-        ]
+        ])
         if self.options.enable_prof:
-            conf_args.append("--enable-prof")
+            tc.configure_args.append("--enable-prof")
         if self.options.shared:
```

Comment:
https://docs.conan.io/2/reference/tools/gnu/autotoolstoolchain.html?highlight=autotoolstoolchain

> Also if the shared option exists it will add by default:
> 
>         --enable-shared, --disable-static if shared==True
>         --disable-shared, --enable-static if shared==False

Reply:
Thanks for pointing that out. Fixed.